src/components/home/index.css

```diff
@@ -34,3 +34,152 @@
   text-align: justify;
   color: #ed2a7a;
 }
+
+.weCare {
+  color: black;
+  margin-left: 150px;
+}
+
+.weCare .pink {
+  color: white;
+  border: none;
+  background-color: deeppink;
+  border-radius: 10px;
+  display: block;
+  font-size: 16px;
+  padding: 12px 28px;
+}
+
+.blue {
+  color: white;
+  border: none;
+  background-color: rgb(59, 175, 241);
+  border-radius: 10px;
+  font-size: 16px;
+  padding: 12px 28px;
+  margin-bottom: 109px;
+}
+
+.weCare h1 {
+  font-family: "Noto Sans";
+  font-size: 46px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 62px;
+  line-height: 73px;
+  text-align: justify;
+  color: #161b50;
+}
+
+.weCare p {
+  width: 100%;
+  text-align: justify;
+  color: #242424;
+  font-family: "Roboto";
+  font-weight: 300;
+  font-size: 24px;
+  padding-right: 70px;
+}
+
+.firstWord {
+  font-size: 36px;
+  width: 400px;
+}
+
+.red {
+  color: deeppink;
+}
+
+.stories {
+  background-color: rgb(230, 245, 244);
+  display: flex;
+  padding-bottom: 100px;
+}
+
+.stories p {
+  text-align: justify;
+  float: left;
+  width: 400px;
+  margin-left: 150px;
+  margin-right: 38px;
+  font-family: "Roboto";
+  font-weight: 300;
+  font-size: 24px;
+}
+
+.stories h1 {
+  margin-left: 150px;
+  margin-top: 102px;
+  width: 403px;
```

Comment:
we probably want to use margins and so on here, generally if you have really big widths and heights and left/top its a bit of a warning sign that probably it should be using percents and margins to align things.